aries-site/src/data/wcag/skeleton.json

```diff
@@ -0,0 +1,513 @@
+[
+    {
+      "rule": "1.1.1",
+      "label": "Non-text content",
+      "status": "failed",
+      "notes": "Skeletons are purely visual and contain no relevant meaning for screen readers so should be given aria-hidden=\"true\". Supporting documentation (https://adrianroselli.com/2020/11/more-accessible-skeletons.html)"
+    },
+    {
+      "rule": "1.2.1",
+      "label": "Audio-only and Video-only (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.2",
+      "label": "Captions (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.3",
+      "label": "Audio Description or Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.4",
+      "label": "Captions (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.5",
+      "label": "Audio Description (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.6",
+      "label": "Sign Language (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.7",
+      "label": "Extended Audio Description (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.8",
+      "label": "Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.9",
+      "label": "Audio-only (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.1",
+      "label": "Info and Relationships",
+      "status": "failed",
+      "notes": "Because skeletons do not have aria-hidden=\"true\" they may mess with the structure of the page for screen readers."
+    },
+    {
+      "rule": "1.3.2",
+      "label": "Meaningful Sequence",
+      "status": "failed",
+      "notes": "Hide skeletons aria-hidden=\"true\" and preserve logical flow."
+    },
+    {
+      "rule": "1.3.3",
+      "label": "Sensory Characteristics",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.4",
+      "label": "Orientation",
+      "status": "passed",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.5",
+      "label": "Identify Input Purpose",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.6",
+      "label": "Identify Purpose",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.1",
+      "label": "Use of Color",
+      "status": "not-applicable",
```

Comment:
would this be applicable in the case we have the boxes with the light grey color loading?

Reply:
The general contrast question is a good one. Skeletons are just placeholders and I'm not convinced they are something required to be visualized all that well. This seems like it needs a bit more research with what others in the industry are doing here.

Reply:
agree I know "use of color" if its something decorative then it doesnt apply so we can do some more research

Reply:
I didn't think that this rule applied, because the skeleton color isn't being used to depict any specific meaning (ex. red = critical).

My interpretation of this rule was that IF color is being used to signify meaning, then there should be an alternative. But in this case, the grey isn't signifying anything.

Reply:
^ Agree, I think 'not-applicable' is correct here